tests/js/spec/views/organizationDashboard/projectCard.spec.jsx

```diff
@@ -7,14 +7,17 @@ describe('ProjectCard', function() {
   let wrapper, projectMock;
   beforeEach(function() {
     wrapper = mount(
-      <ProjectCard project={TestStubs.Project()} params={{orgId: 'org-slug'}} />,
+      <ProjectCard
+        project={TestStubs.Project({platforms: ['javascript']})}
+        params={{orgId: 'org-slug'}}
+      />,
       TestStubs.routerContext()
     );
 
     projectMock = MockApiClient.addMockResponse({
       url: '/projects/org-slug/project-slug/',
       method: 'PUT',
-      data: TestStubs.Project({isBookmarked: false}),
+      data: TestStubs.Project({isBookmarked: false, platforms: ['javascript']}),
```

Comment:
can we add an assertion that `StyledPlatformIcon.platformicon.platformicon-javascript` exists?

Reply:
👍